### DISCUSSION TITLE
Public Dasboards with template variable

### DISCUSSION THREAD

**Discussion by Michsh** [grafana/grafana#64010] [Dashboards, Panels & Visualizations]
HI, i want to make a public dasboard with template variable for selecting data.

Thanks !!!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
